doc/source/workflow.rst

```diff
@@ -0,0 +1,726 @@
+Basic Workflow
+==============
+
+.. topic:: Abstract
+
+    Installation of a Linux system generally occurs by booting the target
+    system from an installation source such as an installation CD/DVD, a live
+    CD/DVD, or a network boot environment (PXE). The installation process is
+    often driven by an installer that interacts with the user to collect
+    information about the installation. This information generally includes the
+    *software to be installed*, the *timezone*, system *user* data, and
+    other information. Once all the information is collected, the installer
+    installs the software onto the target system using packages from the
+    software sources (repositories) available. After the installation is
+    complete the system usually reboots and enters a configuration procedure
+    upon start-up. The configuration may be fully automatic or it may include
+    user interaction.
+    This description applies for version |version|.
+
+A system image (usually called "image"), is a *complete installation* of a Linux
+system within a file. The image represents an operational system and,
+optionally, contains the "final" configuration.
+
+The behavior of the image upon deployment varies depending on the image type
+and the image configuration since KIWI allows you to completely customize
+the initial start-up behavior of the image. Among others, this includes
+"mages that:
```

Comment:
That quote sign is strange here. Probably you mean: "Among others, this includes images that:"